settings.py

```diff
@@ -1914,6 +1914,7 @@ def _determine_couch_databases(couch_databases):
     os.path.join('custom', 'enikshay', 'ucr', 'reports', 'rntcp_pmdt_treatment_register.json'),
     os.path.join('custom', 'enikshay', 'ucr', 'reports', 'referral_report_v2.json'),
     os.path.join('custom', 'enikshay', 'ucr', 'reports', 'drug_voucher.json'),
+    os.path.join('custom', 'enikshay', 'ucr', 'reports', 'dnc_lab_summary.json'),
```

Comment:
why are you adding a report that is not in this PR? was it left out of a different one?

Reply:
@calellowitz 
Was added to the repository several PR earlier (https://github.com/dimagi/commcare-hq/pull/18701). I forgot to add it to the settings.

Reply:
:+1:

Reply:
looks like this should be `dmc_lab_summary.json`

Reply:
:+1: